src/utils/customMessages.js

```diff
@@ -11,7 +11,15 @@ const messages = {
   notExistUser: 'User does not exist',
   changed: 'The password has been changed',
   notChanged: 'The password not changed',
-  errorMessage: 'occured an error'
+  errorMessage: 'occured an error',
+  alreadyExistEmail: 'user with the provided email exist',
```

Comment:
Please change this error to this: The user with the provided email already exists.